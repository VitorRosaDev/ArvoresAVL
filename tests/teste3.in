i 40
i 20
i 60
i 20
r 999
r 40
em
sair