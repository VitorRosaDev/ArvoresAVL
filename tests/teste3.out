OK: inserido 40
Árvore:
└── 40
OK: inserido 20
Árvore:
└── 40
    └── 20
OK: inserido 60
Árvore:
└── 40
    ├── 20
    └── 60
Ignorado: duplicado 20
Árvore:
└── 40
    ├── 20
    └── 60
Ignorado: inexistente 999
Árvore:
└── 40
    ├── 20
    └── 60
OK: removido 40
Árvore:
└── 60
    └── 20
Em-Ordem: 20 60
Encerrado.